ProcessMaker/processmaker
# Implement Template Options for Applying CSS, Layouts, and Fields in Create New Screen

This PR introduces template options for applying CSS, layouts, and fields to newly created screens from a selected template. Users can now customize the application of styling and components by deselecting or selecting options within the 'Preview Template' feature. 

## Changes Made

- Updated the UI for the 'CSS' preview button to standardize with a back arrow button.
- Implemented the ability to create a screen from a template within the 'Preview Template' view.
- Handled selected template options for CSS, layouts, and fields.
- Implemented parsing methods to remove components based on the selected options.
- Added a new ScreenComponent.php file containing arrays of components for CSS, layouts, and fields.

### Layout Components:

- MultiColumn
- Bootstrap Component - Table

### Field Components:
- FormInput
- FormSelectList	
- FormSubmitButton
- FormTextArea
- FormDatePicker
- FormCheckbox
- FormRichText
- FormImage
- FormRecordList
- FormLoop
- FormNestedScreen
- FormPageNavigation
- FileUpload
- FileDownload
- Bootstrap Component - Form Tags
- Bootstrap Component - Embed
- Bootstrap Component - Image
- Bootstrap Component - Form Rating
- Bootstrap Component - Form Spinbutton
- Bootstrap Component - Form Timepicker
- Bootstrap Wrapper - Modal
- Bootstrap Wrapper - Alert
- Bootstrap Wrapper - Card
- Bootstrap Wrapper - Collapse
- Bootstrap Wrapper - Jumbotron
- Photo/Video Control
- Signature Control
- Captcha
- File Preview
- Google Places
- Saved Search Chart


## How to Test

1. Ensure you have a screen template created with custom CSS, layouts, and various fields (refer to the provided list of components).
2. Navigate to Designer > Screens and select '+ Screen' to create a new screen.
3. Choose the created screen template.
4. Click on 'Preview Template'.
5. Test each template option (CSS, Layouts, Fields) by deselecting and selecting them.
6. Create the screen.
7. Verify that the selected template options are properly applied to the new screen, ensuring that CSS, layouts, and fields are included or excluded based on the user's selections.

## Related Tickets & Packages
- [FOUR-13503](https://processmaker.atlassian.net/browse/FOUR-13503)

ci:next

## Code Review Checklist


[FOUR-13503]: https://processmaker.atlassian.net/browse/FOUR-13503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
